client/src/components/OnHover.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+
+class OnHover extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <div border="1px solid width:50px">
+        <ul className="onHov">
+          <li>
+            <svg
+              className="shareReviewIcon"
```

Comment:
Write these in your CSS file. Will make this component a lot smaller and easier to separate the HTML from the CSS

Reply:
@blbst123 resolved